.gitignore

```diff
@@ -0,0 +1,4 @@
+.vscode-test/**
+lib/test/
+coverage/
+node_modules/ 
```

Comment:
I thought node_modules needs to be part of the repo ?

Reply:
yes but i added it for security purposes. All the deps the extension needs are already uploaded here. It may happen that if someone work with this code he/she will end up uploading useless deps and it's quite a waste of time to clean everything again. So i added it to prevent it. We can always remove it later.